exchanges/bitstamp/bitstamp.go

```diff
@@ -86,6 +83,36 @@ func (b *Bitstamp) GetFee(ctx context.Context, feeBuilder *exchange.FeeBuilder)
 	return fee, nil
 }
 
+// GetTradingFee returns a trading fee based on a currency
+func (b *Bitstamp) getTradingFee(ctx context.Context, feeBuilder *exchange.FeeBuilder) (float64, error) {
+	tradingFees, err := b.GetAccountTradingFee(ctx, feeBuilder.Pair)
+	if err != nil {
+		return 0, err
+	}
+	fees := tradingFees.Fees
+	fee := fees.Taker
+	if feeBuilder.IsMaker {
+		fee = fees.Maker
+	}
+	return fee / 100 * feeBuilder.PurchasePrice * feeBuilder.Amount, nil
+}
+
+// GetAccountTradingFee returns a TradingFee for a pair
+func (b *Bitstamp) GetAccountTradingFee(ctx context.Context, pair currency.Pair) (TradingFees, error) {
```

Comment:
If I pass in an empty currency pair it returns a slice, so maybe make a check if the currency pair is empty and return an error. 
```
json: cannot unmarshal array into Go value of type bitstamp.TradingFees
```